docs/RELEASE_NOTES.md

```diff
@@ -23,6 +23,8 @@ If above conditions are met, it is strongly advised to upgrade the MongoDB Kuber
   - The names of connection string secrets generated for configured users are RFC1123 validated.
 - Changes
   - Support for changing port number in running cluster.
+  - Security Context is now defined on pod level (previously was on container level)
+  - Our containers now use the `readOnlyRootFilesystem` setting.
```

Comment:
Perhaps we could mention here, if there is a scenario that will be breaking for someone after upgrading to this version.

Reply:
Actually, the only use case I found for this is the mounting the Ops Manager releases. However, this has been addressed in the enterprise repo. 

Having said that, I think we should be good to go.

Reply:
Q: Do we not have any impact of the change being potential breaking on the init containers for community deployment?

Reply:
No, or at least I could not spot any...

The init container mechanism was working well with this change as we're downloading stuff and write it on the volume (not root filesystem). So apart from some Enterprise ups and downs, I haven't spotted any problem here.